src/components/Dialog/Dialog.md

```diff
@@ -0,0 +1,104 @@
+# Dialog
+Displays content that temporarily blocks interactions with the application.
```

Comment:
I would include: "It is primarily used for confirmation tasks, light-weight creation/edit tasks, and simple management tasks."